test/store.spec.js

```diff
@@ -93,5 +94,109 @@ describe('Store Tests', function() {
 			done();
 		}).catch(done);
 	});
+
+	it('Store.prototype._filter() should return a Promise rejected', function(done) {
+		store._filter().catch(function(err) {
+			err.message.should.equal('Store.prototype._filter(model, query) is not implemented');
+			done();
+		});
+	});
+
+	it('Store.prototype._get() should return a Promise rejected', function(done) {
+		store._get().catch(function(err) {
+			err.message.should.equal('Store.prototype._get(model, pk) is not implemented');
+			done();
+		});
+	});
+
+	it('Store.prototype._set() should return a Promise rejected', function(done) {
+		store._set().catch(function(err) {
+			err.message.should.equal('Store.prototype._set(model, obj, operation) is not implemented');
+			done();
+		});
+	});
+
+	it('Store.prototype._delete() should return a Promise rejected', function(done) {
+		store._delete().catch(function(err) {
+			err.message.should.equal('Store.prototype._delete(query) is not implemented');
+			done();
+		});
+	});
+
+	describe('Cache Integration should', function() {
```

Comment:
Here I am actually testing the integration of cache in store and not the actual cache implementation as it does not exist.

I think it make more sense for this to be here.